### PR TITLE
Added organizer info (Liling)

### PR DESCRIPTION
**Pull request by alvations**

### CODE REVIEW

File: Proposal.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ To promote closer interaction, all accepted publications will be presented in th
  - [Masato Hagiwara](http://masatohagiwara.net/), Duolingo Inc.
  - [Dmitrijs Milajevs](http://www.eecs.qmul.ac.uk/~dm303/), NIST and Queen Mary University of London
  - [Liling Tan](https://github.com/alvations), Rakuten Institute of Technology
+ 
+      Liling is a research scientist at Rakuten Institute of Technology working on Machine Translation and developing applications using language technologies. He has been actively involved in corpora creation/maintenance, Asian NLP and machine translation. He co-organized the [Workshop on NLP for Similar Languages, Varieties  and Dialects (VarDial 2014-16)](http://ttg.uni-saarland.de/vardial2016/) and the DSL shared task.
```
Comment (mhagiwara) [inline Proposal.md]: There's a weird space between `Varieties` and `and`

**Comment by alvations** [comment]
#9

**Comment by mhagiwara** [comment]
LGTM

**Comment by e9t** [comment]
LGTM 👍

**Comment by dimazest** [comment]
Looks good.

**Pull request merged**